crates/biome_cli/src/execute/migrate/prettier.rs

```diff
@@ -71,10 +76,46 @@ impl Default for PrettierConfiguration {
             jsx_single_quote: false,
             arrow_parens: ArrowParens::default(),
             end_of_line: EndOfLine::default(),
+            overrides: vec![],
         }
     }
 }
 
+#[derive(Clone, Debug, Default, Deserializable, Eq, PartialEq)]
+pub(crate) struct Override {
+    files: IndexSet<String>,
+    options: OverrideOptions,
+}
+
+#[derive(Clone, Debug, Default, Deserializable, Eq, PartialEq)]
```

Comment:
Out of curiosity, can we use our `Partial` macro to save all those `Option`?

Reply:
You mean using `Partial` on `PrettierConfiguration`?
Unfortunately `PrettierConfiguration` has an extra field `overrides`.
Thus, we cannot do that.

If you mean removing `Option` on every field and using `Partial`, I don't see the interest of doing that because we have no need of this type without the `Option`.